public/locations-list-map.js

```diff
@@ -358,6 +358,32 @@ if (window.HTMLCollection && !HTMLCollection.prototype.forEach) {
   HTMLCollection.prototype.forEach = Array.prototype.forEach;
 }
 
+// Lazy-loads the Google maps script once we know we need it. Sets up
+// a global initMap callback on the window object so the gmap script
+// can find it.
+function loadMapScript(searchParams, stateFilter) {
+  // Property created on window must match name passed in &callback= param
+  window.initMap = () => initMap(stateFilter);
+
+  // load map based on current lang
+  const scriptTag = ce('script');
+
+  // API Key below is only enabled for *.findthemasks.com/* Message @susanashlock for more info.
+  const apiKey = 'AIzaSyDSz0lnzPJIFeWM7SpSARHmV-snwrAXd2s';
+  let scriptSrc = `//maps.googleapis.com/maps/api/js?libraries=geometry,places&callback=initMap&key=${ apiKey }`;
```

Comment:
Woo, awesome. Now we have the opportunity to improve upon this further by potentially pulling `apiKey` from the query string and substituting it here prior to the `<script>` tag being appended to `<body>`.